documentation/api/io.strimzi.api.kafka.model.KafkaMirrorMaker2ClusterSpec.adoc

```diff
@@ -0,0 +1,18 @@
+Use the three allowed `ssl` configuration options to run external listeners on a specific _cipher suite_ for a TLS version.
```

Comment:
is it "on a specific cipher suite" the right one? or should it be "with a specific cipher suite"? Same comment for all the other files.

Reply:
_with_ is better